app/src/main/java/com/simplecity/amp_library/utils/SettingsManager.java

```diff
@@ -22,6 +22,11 @@ public SettingsManager(@NotNull SharedPreferences sharedPreferences) {
         super(sharedPreferences);
     }
 
+    private static SettingsManager instance;
+
+    public static SettingsManager getInstance() {
```

Comment:
How was this class accessed previously? I want to make sure we're not duplicating access and injecting via dependency injection framework and also accessing a static instance.

Again, if this is a static instance accessed by multiple threads this method should probably be `synchronized`.

Reply:
It was never accessed before in the MUSER. It's a new addition since `DialogUtils` required that function to be created.